### PR TITLE
Added serv.warn() & other small changes

### PR DESCRIPTION
**Pull request by andriycraft**
Added serv.warn() & other small changes

### CODE REVIEW

**Comment by rom1504** [comment]
ci failing

File: src/lib/plugins/log.js [hunk 1]
```diff
@@ -23,8 +23,6 @@ module.exports.server = function (serv, settings) {
   serv.on('banned', (banner, bannedUsername, reason) =>
     serv.info(banner.username + ' banned ' + bannedUsername + (reason ? ' (' + reason + ')' : '')))
 
-  serv.on('seed', (seed) => serv.log('seed: ' + seed))
```
Comment (rom1504) [inline src/lib/plugins/log.js:26]: Why did you remove this ?

File: src/lib/plugins/log.js [hunk 2]
```diff
@@ -42,7 +40,11 @@ module.exports.server = function (serv, settings) {
   }
 
   serv.err = message => {
-    serv.log('[' + colors.red('ERR') + ']: ' + message)
+    serv.log('[' + colors.red('ERROR') + ']: ' + message)
+  }
+
+  serv.warn = message => {
```
Comment (rom1504) [inline src/lib/plugins/log.js:46]: Add it to the doc

Is is useful? If yes, use it at some places in flying squid

File: src/lib/plugins/log.js (hunk 1 above)
Comment (andriycraft) [reply src/lib/plugins/log.js:26]: Is is useful?

File: src/lib/plugins/log.js (hunk 2 above)
Comment (andriycraft) [reply src/lib/plugins/log.js:46]: done

File: src/lib/plugins/log.js (hunk 1 above)
Comment (rom1504) [reply src/lib/plugins/log.js:26]: yes

**Comment by rom1504** [comment]
CI is failing

**Comment by andriycraft** [comment]
Done.
